netsim/ansible/templates/vxlan/frr.j2

```diff
@@ -13,11 +13,12 @@ ip link add vxlan{{ vni }} type vxlan \
 #
 # Add it to the VLAN bridge (create if needed for l3 vnis); disable STP
 if [ ! -e /sys/devices/virtual/net/{{ br_name}} ]; then
-brctl addbr {{ br_name }}
+ip link add {{ br_name }} type bridge
 ip link set up dev {{ br_name }}
 fi
-brctl addif {{ br_name }} vxlan{{ vni }}
-brctl stp {{ br_name }} off
+ip link set dev vxlan{{ vni }} master {{ br_name }}
+ip link set {{ br_name }} type bridge stp_state 1
```

Comment:
The original code **disabled** STP on VXLAN bridges, while your code **enables** it. Is there a reason for this change or is it a typo?

Reply:
Typo - will fix